redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -40,7 +40,7 @@
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = DatabaseServerOpDescription.LIST_BY_WORKSPACE, produces = ContentType.JSON, notes = Notes.DATABASE_SERVER_NOTES,
         nickname = "listDatabasesServersByWorkspace")
-    DatabaseServerV4Responses list(@PathParam("workspaceId") Long workspaceId, @QueryParam("environment") String environment,
+    DatabaseServerV4Responses list(@PathParam("workspaceId") Long workspaceId, @QueryParam("environmentId") Long environmentIds,
```

Comment:
environment id is a CRN - this change, however, can be part of another PR if you think it is better to do things more fine-grained.

Reply:
I can definitely update it now.